suppressible-error-prone-annotations/build.gradle

```diff
@@ -0,0 +1,2 @@
+apply plugin: 'java-library'
```

Comment:
is this a separate project so that things outside of this repo can use the annotations if desired?  Or is it needed to add to the classpath of the target project itself when running the second pass?

Reply:
The latter, [see here](https://github.com/palantir/suppressible-error-prone/blob/ee8fced1def59cbf7c37b414ae4c5fe4ea43e4f8/gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java#L74-L79)